### ISSUE TITLE
Creation of a GUI

### ISSUE THREAD

**Issue by b-regan11** [omarf72/CSCchallenge1#9]
Create a proper graphical user interface for the store and the operations within the store.
Currently, the terminal is the only way of navigating through the store. This is functionally fine but visually less than.

**Comment by omarf72** [comment]
I agree with making a proper user interface to make the store easier to navigate for the user. There could be more visuals and less words to make the store easier to use.
